todo_app.js

```diff
@@ -63,15 +75,28 @@
          }*/
     }
 
-    function renderTodo(todoObject) {
+    function renderTodos(list) {
```

Comment:
you should select:
retrieve todos inside current method or pass them via arguments. Now you're not using passed todos and retrieve list again inside the method.